adapters/InMobi/InMobiAdapter/InMobiMediatedNativeAppInstallAd.m

```diff
@@ -64,22 +64,38 @@ - (instancetype)initWithInMobiNativeAppInstallAd:(IMNative *)nativeAd
       NSURL *imageURL = [NSURL URLWithString:imageStringURL];
       NSString *iconStringURL = [iconDictionary objectForKey:URL];
       NSURL *iconURL = [NSURL URLWithString:iconStringURL];
-
-      if (!shouldDownloadImage) {
-        SEL inmobiMediatedNativeAppInstallAdSuccessful =
-            @selector(inmobiMediatedNativeAppInstallAdSuccessful:);
-        self.mappedImages = @[
-          [[GADNativeAdImage alloc]
-              initWithURL:imageURL
-                    scale:[[imageDictionary objectForKey:ASPECT_RATIO] floatValue]]
-        ];
-        self.mappedIcon = [[GADNativeAdImage alloc]
-            initWithURL:iconURL
-                  scale:[[iconDictionary objectForKey:ASPECT_RATIO] floatValue]];
-        if ([self respondsToSelector:inmobiMediatedNativeAppInstallAdSuccessful]) {
-          [self inmobiMediatedNativeAppInstallAdSuccessful:self];
-        }
-      } else {
+        CGFloat imageScale;
+        CGFloat iconScale;
+        if(!shouldDownloadImage){
+            SEL inmobiMediatedNativeAppInstallAdSuccessful = @selector(inmobiMediatedNativeAppInstallAdSuccessful:);
+            if([imageDictionary objectForKey:ASPECT_RATIO]){
```

Comment:
Note that the scale is the [scale factor](https://developer.apple.com/ios/human-interface-guidelines/icons-and-images/image-size-and-resolution/) for the image resolution (eg 1x, 2x, or 3x) and not the Aspect Ratio. If the InMobi SDK does not have and API for Scale, the adapter should just return `1` for scale.